src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -439,6 +461,10 @@ function WorkspacesListPage() {
                 cancelText={translate('common.cancel')}
                 danger
             />
+            <SupportalActionModal
```

Comment:
What do y'all think about calling this `SupportalActionRestrictedModal` or `RestrictedSupportalActionModal` to make it super clear that this modal specifically tells the user that an action is **restricted** since they're supportaled?